src/shared/components/rightbar/RightBar.tsx

```diff
@@ -303,9 +304,31 @@ export default class RightBar extends React.Component<
         ) : null;
     }
 
+    getHomePageTour() {
+        const isLoading =
+            !this.studyStore.cancerTypes.isComplete ||
+            !this.studyStore.cancerStudies.isComplete ||
+            !this.studyStore.userVirtualStudies.isComplete;
+        return getServerConfig().skin_right_nav_show_testimonials ? (
```

Comment:
maybe we can add another property for tours specifically?

Reply:
Thanks for reminder!! I'll add one.